packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -802,7 +802,7 @@ class _Label extends StatelessWidget {
 
 class _BottomNavigationBarState extends State<BottomNavigationBar> with TickerProviderStateMixin {
   List<AnimationController> _controllers = <AnimationController>[];
-  late List<CurvedAnimation> _animations;
+  List<CurvedAnimation> _animations = const <CurvedAnimation>[];
 
```

Comment:
This `const` is alerting for me.
It may happen that in future changes we will want to add items to the list, but if it is const, it will fail. 
I suggest to remove it.

Reply:
Sure I'll do that

Reply:
I removed the `const` keyword in [refactor: Remove const](https://github.com/flutter/flutter/pull/147213/commits/8643eb5988ae66a5ae8e089ccde93f9f4e3ed731)